### ISSUE TITLE
Add keyboard interaction model for new basic color picker

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#8657]
## Feature Description

The new basic color picker should be easier to use when using a keyboard-only. Ensure full a11y compliance including

- Keyboard navigation 
- Focus styling
- Labels

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Verified in QA

**Comment by bmattb** [comment]
Can we close this @barklund ?
